lib/Models/Catalog/Ows/WebMapTileServiceCapabilities.ts

```diff
@@ -145,45 +128,55 @@ export default class WebMapTileServiceCapabilities {
           });
         }
 
-        return new WebMapTileServiceCapabilities(capabilitiesXml, json);
+        console.log(json);
```

Comment:
There are a couple of `console.log()` calls, are they forgotten leftovers in this PR as well?